test/integration/test_events.py

```diff
@@ -125,7 +125,7 @@ def test_playbook_on_stats_summary_fields(project_fixtures):
         assert runner_stats[stat]  # expected at least 1 host in each stat type
 
 
-def test_include_role_events(project_fixtures):
+def test_include_role_events(project_fixtures, skipif_ansible_219_or_higher):  # pylint: disable=W0613
```

Comment:
It's usually nicer to have a `@pytest.mark.usefixtures('skipif_ansible_219_or_higher')` for unused arguments as it will make it clear what you actually access in the test function + you wouldn't have to deal with pointless suppressions. Additionally, I wanted to call out that suppressions with rule numbers in pylint aren't easy to follow and it's best to use human-readable identifiers instead.